cmd/api/database/database.go

```diff
@@ -92,8 +106,42 @@ func countRows(name string, minPrice32, maxPrice32 float32, archived bool) (int,
 	return count, nil
 }
 
+/* Stores the book into the database, checks and returns it if succeed. */
+func (store *Store) CreateBook(bookEntry book.Book) (book.Book, error) {
+	sqlStatement := `
+	INSERT INTO bookstable (id, name, price, inventory, created_at, updated_at)
+	VALUES ($1, $2, $3, $4, $5, $6)
+	RETURNING *`
+	createdRow := store.db.QueryRow(sqlStatement, bookEntry.ID, bookEntry.Name, *bookEntry.Price, *bookEntry.Inventory, bookEntry.CreatedAt, bookEntry.UpdatedAt)
+	var bookToReturn book.Book
+	err := createdRow.Scan(&bookToReturn.ID, &bookToReturn.Name, &bookToReturn.Price, &bookToReturn.Inventory, &bookToReturn.CreatedAt, &bookToReturn.UpdatedAt, &bookToReturn.Archived)
+	if err != nil {
+		return book.Book{}, fmt.Errorf("storing on db: %w", err)
+	}
+
+	return bookToReturn, nil
+}
+
+/* Searches a book in database based on ID and returns it if succeed. */
+func (store *Store) GetBookByID(id uuid.UUID) (book.Book, error) {
+	sqlStatement := `SELECT id, name, price, inventory, created_at, updated_at, archived FROM bookstable WHERE id=$1;`
```

Comment:
Please format this `sql` query like you did with the others. They read way better.

Reply:
ok, done.